app/grandchallenge/core/templates/challenge.html

```diff
@@ -33,79 +33,78 @@
-                    {% if challenge.use_teams %}
-                        {% if "change_challenge" in challenge_perms or user_is_participant %}
-                            <li class="nav-item">
-                                <a class="nav-link {% if request.resolver_match.view_name == 'teams:list' %}active{% endif %}"
-                                   href="{% url 'teams:list' challenge_short_name=challenge.short_name %}">Teams</a>
-                            </li>
+                    {% if challenge.use_evaluation %}
+                        {% if challenge.use_teams %}
+                            {% if "change_challenge" in challenge_perms or user_is_participant %}
+                                <li class="nav-item">
+                                    <a class="nav-link {% if request.resolver_match.view_name == 'teams:list' %}active{% endif %}"
+                                       href="{% url 'teams:list' challenge_short_name=challenge.short_name %}">Teams</a>
+                                </li>
+                            {% endif %}
                         {% endif %}
-                    {% endif %}
-                    {% for phase in challenge.phase_set.all %}
+
                         {% if "change_challenge" in challenge_perms or user_is_participant %}
                             <li class="nav-item">
-                                <a class="nav-link {% if request.resolver_match.view_name == 'evaluation:submission-create' and request.resolver_match.kwargs.slug == phase.slug %}active{% endif %}"
-                                   href="{% url 'evaluation:submission-create' challenge_short_name=challenge.short_name slug=phase.slug %}">Create {{ phase.title }}
-                                    Submission</a>
+                                <a class="nav-link {% if request.resolver_match.view_name == 'evaluation:submission-create' %}active{% endif %}"
+                                   href="{% url 'evaluation:submission-create' challenge_short_name=challenge.short_name slug=challenge.phase_set.first.slug %}">
+                                    Submit</a>
                             </li>
                             <li class="nav-item">
-                                <a class="nav-link {% if request.resolver_match.view_name == 'evaluation:leaderboard' and request.resolver_match.kwargs.slug == phase.slug %}active{% endif %}"
-                                   href="{% url 'evaluation:leaderboard' challenge_short_name=challenge.short_name slug=phase.slug %}">{{ phase.title }}
-                                    Leaderboard</a>
+                                <a class="nav-link {% if request.resolver_match.view_name == 'evaluation:leaderboard' %}active{% endif %}"
+                                   href="{% url 'evaluation:leaderboard' challenge_short_name=challenge.short_name slug=challenge.phase_set.first.slug %}">
+                                    Leaderboard(s)</a>
```

Comment:
I think the plural can be removed here. If you really want to pluraise it then it is cleaner to use the built in template tag https://docs.djangoproject.com/en/3.1/ref/templates/builtins/#pluralize

Reply:
Ah, you're right, but just for good practice for me. If I were to pluralize, it would be `Leaderboard{{ challenge.phase_set.all|pluralize }}` - correct?

Reply:
Spot on

Reply:
It seems to work and I kinda like that it changes, so I might keep it in there - but of course I can always take it out if you'd rather not have this

Reply:
Check that it does not make another db query, I think that it does. See this section for how to optimise that https://docs.djangoproject.com/en/3.1/topics/db/optimization/#don-t-overuse-count-and-exists